pyproject.toml

```diff
@@ -3,9 +3,23 @@ requires = ["setuptools"]
 
 [tool.cibuildwheel]
 build-verbosity = 2
-skip = ["pp*", "gp*", "cp31?t-*", "cp36*", "cp37*"]
+skip = [
+    "pp*", "gp*", "cp31?t-*", "cp36*", "cp37*",
+    # The platforms below are non-core platforms that we're providing
+    # Stable ABI (>3.9) wheels to. Python 3.8 is excluded and will use
+    # pure-Python wheels. cibuildwheel will identify Stable ABI wheels and
+    # only build them for the one Python version. It will test them for all
+    # Python versions though.
```

Comment:
Despite what [the documentation says](https://cibuildwheel.pypa.io/en/stable/changelog/#v250), this doesn't appear to be true in practice - it's building multiple versions of the same thing.

Reply:
It may be the `only` option that's stopping it from working. 

I also don't know why I seem to have broken the 3.13t tests (which should be unaffected)